### ISSUE TITLE
"brew install libpqxx" fails

### ISSUE THREAD

**Issue by xandrrix** [Linuxbrew/homebrew-core#5112]
So below I've output what happened. Basically this is what happens: 
1. Attempt to install libpqxx.
2. Told 

> *****************************************************
> 
> xmlto not found.
> Install it, or configure with --disable-documentation
> 
> *****************************************************
> Makefile:485: recipe for target 'reference-stamp' failed
> make[1]: *** [reference-stamp] Error 1
> make[1]: *** Waiting for unfinished jobs....
> Makefile:505: recipe for target 'tutorial-stamp' failed
> make[1]: *** [tutorial-stamp] Error 1
> make[1]: Leaving directory '/tmp/libpqxx-20171208-86877-1smohyp/libpqxx-5.0.1/doc'
> Makefile:684: recipe for target 'install-recursive' failed
> make: *** [install-recursive] Error 1

3. Install xmlto "manually" with brew install xmoto
4. Run "brew install libpqxx" again. 
5. Same Error. :-/

SOLUTION: add " --disable-documentation" as it suggests somehow? 

xandrrix@xandrrix-virtual-machine:~$ brew install libpqxx
==> Tapping linuxbrew/extra
Cloning into '/home/xandrrix/.linuxbrew/Library/Taps/linuxbrew/homebrew-extra'...
remote: Counting objects: 18, done.
remote: Compressing objects: 100% (17/17), done.
remote: Total 18 (delta 0), reused 6 (delta 0), pack-reused 0
Unpacking objects: 100% (18/18), done.
Checking connectivity... done.
Tapped 5 formulae (52 files, 196.9KB)
==> Installing dependencies for libpqxx: xz, ncurses, readline, sqlite, gdbm, berkeley-db@4, unzip, python, libxml2, libxslt, berkeley-db, libbsd, expat, perl, util-linux, postgresql
==> Installing libpqxx dependency: xz
==> Downloading https://linuxbrew.bintray.com/bottles/xz-5.2.3.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Pouring xz-5.2.3.x86_64_linux.bottle.tar.gz
🍺  /home/xandrrix/.linuxbrew/Cellar/xz/5.2.3: 99 files, 1.8MB
==> Installing libpqxx dependency: ncurses
==> Downloading https://ftp.gnu.org/gnu/ncurses/ncurses-6.0.tar.gz
######################################################################## 100.0%
==> Downloading https://invisible-mirror.net/archives/ncurses/6.0/ncurses-6.0-20170930-patch.sh.bz2
######################################################################## 100.0%
==> sh ncurses-6.0-20170930-patch.sh
==> ./configure --prefix=/home/xandrrix/.linuxbrew/Cellar/ncurses/6.0_4 --enable-pc-files --with-pkg-config-libdir=/home/xandrrix/.linuxbrew/Cellar/ncurses/6.0_4/lib/pkgconfig --e
==> make
==> make install
🍺  /home/xandrrix/.linuxbrew/Cellar/ncurses/6.0_4: 3,899 files, 17.1MB, built in 2 minutes 5 seconds
==> Installing libpqxx dependency: readline
==> Downloading https://linuxbrew.bintray.com/bottles/readline-7.0.3_1.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Pouring readline-7.0.3_1.x86_64_linux.bottle.tar.gz
🍺  /home/xandrrix/.linuxbrew/Cellar/readline/7.0.3_1: 46 files, 1.7MB
==> Installing libpqxx dependency: sqlite
==> Downloading https://linuxbrew.bintray.com/bottles/sqlite-3.21.0.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Pouring sqlite-3.21.0.x86_64_linux.bottle.tar.gz
🍺  /home/xandrrix/.linuxbrew/Cellar/sqlite/3.21.0: 12 files, 19.5MB
==> Installing libpqxx dependency: gdbm
==> Downloading https://linuxbrew.bintray.com/bottles/gdbm-1.13_1.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Pouring gdbm-1.13_1.x86_64_linux.bottle.tar.gz
🍺  /home/xandrrix/.linuxbrew/Cellar/gdbm/1.13_1: 37 files, 2MB
==> Installing libpqxx dependency: berkeley-db@4
==> Downloading https://linuxbrew.bintray.com/bottles/berkeley-db@4-4.8.30.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Pouring berkeley-db@4-4.8.30.x86_64_linux.bottle.tar.gz
==> Caveats
This formula is keg-only, which means it was not symlinked into /home/xandrrix/.linuxbrew,
because this is an alternate version of another formula.

If you need to have this software first in your PATH run:
  echo 'export PATH="/home/xandrrix/.linuxbrew/opt/berkeley-db@4/bin:$PATH"' >> ~/.bash_profile

For compilers to find this software you may need to set:
    LDFLAGS:  -L/home/xandrrix/.linuxbrew/opt/berkeley-db@4/lib
    CPPFLAGS: -I/home/xandrrix/.linuxbrew/opt/berkeley-db@4/include

==> Summary
🍺  /home/xandrrix/.linuxbrew/Cellar/berkeley-db@4/4.8.30: 4,596 files, 82.2MB
==> Installing libpqxx dependency: unzip
==> Downloading https://linuxbrew.bintray.com/bottles/unzip-6.0_3.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Pouring unzip-6.0_3.x86_64_linux.bottle.tar.gz
🍺  /home/xandrrix/.linuxbrew/Cellar/unzip/6.0_3: 15 files, 434.5KB
==> Installing libpqxx dependency: python
==> Downloading https://www.python.org/ftp/python/2.7.14/Python-2.7.14.tar.xz
######################################################################## 100.0%
==> ./configure --prefix=/home/xandrrix/.linuxbrew/Cellar/python/2.7.14 --enable-ipv6 --datarootdir=/home/xandrrix/.linuxbrew/Cellar/python/2.7.14/share --datadir=/home/xandrrix/.
==> make
==> make install PYTHONAPPSDIR=/home/xandrrix/.linuxbrew/Cellar/python/2.7.14
==> Downloading https://files.pythonhosted.org/packages/a4/c8/9a7a47f683d54d83f648d37c3e180317f80dc126a304c45dc6663246233a/setuptools-36.5.0.zip
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/11/b6/abcb525026a4be042b486df43905d6893fb04f05aac21c32c638e939e447/pip-9.0.1.tar.gz
######################################################################## 100.0%
==> Downloading https://files.pythonhosted.org/packages/fa/b4/f9886517624a4dcb81a1d766f68034344b7565db69f13d52697222daeb72/wheel-0.30.0.tar.gz
######################################################################## 100.0%
==> /home/xandrrix/.linuxbrew/Cellar/python/2.7.14/bin/python2 -s setup.py --no-user-cfg install --force --verbose --single-version-externally-managed --record=installed.txt --ins
==> /home/xandrrix/.linuxbrew/Cellar/python/2.7.14/bin/python2 -s setup.py --no-user-cfg install --force --verbose --single-version-externally-managed --record=installed.txt --ins
==> /home/xandrrix/.linuxbrew/Cellar/python/2.7.14/bin/python2 -s setup.py --no-user-cfg install --force --verbose --single-version-externally-managed --record=installed.txt --ins
==> Caveats
This formula installs a python2 executable to /home/xandrrix/.linuxbrew/bin.
If you wish to have this formula's python executable in your PATH then add
the following to ~/.bash_profile:
  export PATH="/home/xandrrix/.linuxbrew/opt/python/libexec/bin:$PATH"

Pip and setuptools have been installed. To update them
  pip2 install --upgrade pip setuptools

You can install Python packages with
  pip2 install <package>

They will install into the site-package directory
  /home/xandrrix/.linuxbrew/lib/python2.7/site-packages

See: https://docs.brew.sh/Homebrew-and-Python.html

Python modules have been installed and Homebrew's site-packages is not
in your Python sys.path, so you will not be able to import the modules
this formula installed. If you plan to develop with these modules,
please run:
  mkdir -p /home/xandrrix/.local/lib/python2.7/site-packages
  echo 'import site; site.addsitedir("/home/xandrrix/.linuxbrew/lib/python2.7/site-packages")' >> /home/xandrrix/.local/lib/python2.7/site-packages/homebrew.pth
==> Summary
🍺  /home/xandrrix/.linuxbrew/Cellar/python/2.7.14: 5,490 files, 105.8MB, built in 2 minutes 55 seconds
==> Installing libpqxx dependency: libxml2
==> Downloading https://linuxbrew.bintray.com/bottles/libxml2-2.9.7.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Pouring libxml2-2.9.7.x86_64_linux.bottle.tar.gz
==> Caveats
Python modules have been installed and Homebrew's site-packages is not
in your Python sys.path, so you will not be able to import the modules
this formula installed. If you plan to develop with these modules,
please run:
  mkdir -p /home/xandrrix/.local/lib/python2.7/site-packages
  echo 'import site; site.addsitedir("/home/xandrrix/.linuxbrew/lib/python2.7/site-packages")' >> /home/xandrrix/.local/lib/python2.7/site-packages/homebrew.pth
==> Summary
🍺  /home/xandrrix/.linuxbrew/Cellar/libxml2/2.9.7: 282 files, 23.6MB
==> Installing libpqxx dependency: libxslt
==> Downloading http://xmlsoft.org/sources/libxslt-1.1.32.tar.gz
######################################################################## 100.0%
==> ./configure --disable-silent-rules --prefix=/home/xandrrix/.linuxbrew/Cellar/libxslt/1.1.32 --without-crypto --with-libxml-prefix=/home/xandrrix/.linuxbrew/opt/libxml2
==> make
==> make install
==> Caveats
To allow the nokogiri gem to link against this libxslt run:
  gem install nokogiri -- --with-xslt-dir=/home/xandrrix/.linuxbrew/opt/libxslt
==> Summary
🍺  /home/xandrrix/.linuxbrew/Cellar/libxslt/1.1.32: 139 files, 6MB, built in 34 seconds
==> Installing libpqxx dependency: berkeley-db
==> Downloading https://linuxbrew.bintray.com/bottles/berkeley-db-6.2.32.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Pouring berkeley-db-6.2.32.x86_64_linux.bottle.tar.gz
🍺  /home/xandrrix/.linuxbrew/Cellar/berkeley-db/6.2.32: 5,632 files, 128.3MB
==> Installing libpqxx dependency: libbsd
==> Downloading https://linuxbrew.bintray.com/bottles/libbsd-0.8.3.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Pouring libbsd-0.8.3.x86_64_linux.bottle.tar.gz
🍺  /home/xandrrix/.linuxbrew/Cellar/libbsd/0.8.3: 82 files, 713.9KB
==> Installing libpqxx dependency: expat
==> Downloading https://linuxbrew.bintray.com/bottles/expat-2.2.5.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Pouring expat-2.2.5.x86_64_linux.bottle.tar.gz
🍺  /home/xandrrix/.linuxbrew/Cellar/expat/2.2.5: 18 files, 1.8MB
==> Installing libpqxx dependency: perl
==> Downloading https://www.cpan.org/src/5.0/perl-5.26.1.tar.xz
######################################################################## 100.0%
==> ./Configure -des -Dprefix=/home/xandrrix/.linuxbrew/Cellar/perl/5.26.1 -Dprivlib=/home/xandrrix/.linuxbrew/Cellar/perl/5.26.1/lib/perl5/5.26.1 -Dsitelib=/home/xandrrix/.linuxb
==> make
==> make install
==> /home/xandrrix/.linuxbrew/Cellar/perl/5.26.1/bin/cpan -i XML::Parser
==> /home/xandrrix/.linuxbrew/Cellar/perl/5.26.1/bin/cpan -i XML::SAX
==> Caveats
By default non-brewed cpan modules are installed to the Cellar. If you wish
for your modules to persist across updates we recommend using `local::lib`.

You can set that up like this:
  PERL_MM_OPT="INSTALL_BASE=$HOME/perl5" cpan local::lib
  echo 'eval "$(perl -I$HOME/perl5/lib/perl5 -Mlocal::lib)"' >> ~/.bash_profile
==> Summary
🍺  /home/xandrrix/.linuxbrew/Cellar/perl/5.26.1: 2,730 files, 67.9MB, built in 6 minutes 19 seconds
==> Installing libpqxx dependency: util-linux
==> Downloading https://www.kernel.org/pub/linux/utils/util-linux/v2.30/util-linux-2.30.1.tar.xz
######################################################################## 100.0%
==> ./configure --disable-silent-rules --prefix=/home/xandrrix/.linuxbrew/Cellar/util-linux/2.30.1 --disable-use-tty-group --disable-kill --disable-cal --disable-ul --without-syst
==> make install
==> Caveats
Some commands were moved to the bsdmainutils formula.
You may wish to install it as well.

Bash completion has been installed to:
  /home/xandrrix/.linuxbrew/etc/bash_completion.d
==> Summary
🍺  /home/xandrrix/.linuxbrew/Cellar/util-linux/2.30.1: 356 files, 33.6MB, built in 2 minutes 20 seconds
==> Installing libpqxx dependency: postgresql
==> Downloading https://ftp.postgresql.org/pub/source/v10.1/postgresql-10.1.tar.bz2
######################################################################## 100.0%
==> ./configure --prefix=/home/xandrrix/.linuxbrew/Cellar/postgresql/10.1 --datadir=/home/xandrrix/.linuxbrew/share/postgresql --libdir=/home/xandrrix/.linuxbrew/lib --sysconfdir=
==> make
==> make install-world datadir=/home/xandrrix/.linuxbrew/Cellar/postgresql/10.1/share/postgresql libdir=/home/xandrrix/.linuxbrew/Cellar/postgresql/10.1/lib pkglibdir=/home/xandrr
==> /home/xandrrix/.linuxbrew/Cellar/postgresql/10.1/bin/initdb /home/xandrrix/.linuxbrew/var/postgres
==> Caveats
To migrate existing data from a previous major version of PostgreSQL, see:
  https://www.postgresql.org/docs/10/static/upgrading.html

  You will need your previous PostgreSQL installation from brew to perform
  `pg_upgrade` or `pg_dumpall` depending on your upgrade method.

  Do not run `brew cleanup postgresql` until you have performed the migration.
==> Summary
🍺  /home/xandrrix/.linuxbrew/Cellar/postgresql/10.1: 3,364 files, 42.3MB, built in 3 minutes 51 seconds
==> Installing libpqxx
==> Downloading https://github.com/jtv/libpqxx/archive/5.0.1.tar.gz
==> Downloading from https://codeload.github.com/jtv/libpqxx/tar.gz/5.0.1
######################################################################## 100.0%
==> ./configure --prefix=/home/xandrrix/.linuxbrew/Cellar/libpqxx/5.0.1_1 --enable-shared
==> make install
Last 15 lines from /home/xandrrix/.cache/Homebrew/Logs/libpqxx/02.make:

*****************************************************

xmlto not found.
Install it, or configure with --disable-documentation

Makefile:485: recipe for target 'reference-stamp' failed
*****************************************************
make[1]: *** [reference-stamp] Error 1
make[1]: *** Waiting for unfinished jobs....
Makefile:505: recipe for target 'tutorial-stamp' failed
make[1]: *** [tutorial-stamp] Error 1
make[1]: Leaving directory '/tmp/libpqxx-20171208-63303-89jbov/libpqxx-5.0.1/doc'
Makefile:684: recipe for target 'install-recursive' failed
make: *** [install-recursive] Error 1

READ THIS: https://github.com/Linuxbrew/brew/blob/master/docs/Troubleshooting.md#troubleshooting
Please do not report this issue to Homebrew/brew or Homebrew/core, which support macOS only.

xandrrix@xandrrix-virtual-machine:~$ brew install libpqxx
==> Downloading https://github.com/jtv/libpqxx/archive/5.0.1.tar.gz
Already downloaded: /home/xandrrix/.cache/Homebrew/libpqxx-5.0.1.tar.gz
==> ./configure --prefix=/home/xandrrix/.linuxbrew/Cellar/libpqxx/5.0.1_1 --enable-shared
==> make install
Last 15 lines from /home/xandrrix/.cache/Homebrew/Logs/libpqxx/02.make:
Makefile:485: recipe for target 'reference-stamp' failed

make[1]: *** [reference-stamp] Error 1
make[1]: *** Waiting for unfinished jobs....
*****************************************************

xmlto not found.
Install it, or configure with --disable-documentation

*****************************************************
Makefile:505: recipe for target 'tutorial-stamp' failed
make[1]: *** [tutorial-stamp] Error 1
make[1]: Leaving directory '/tmp/libpqxx-20171208-68242-fy26fz/libpqxx-5.0.1/doc'
Makefile:684: recipe for target 'install-recursive' failed
make: *** [install-recursive] Error 1

READ THIS: https://github.com/Linuxbrew/brew/blob/master/docs/Troubleshooting.md#troubleshooting
Please do not report this issue to Homebrew/brew or Homebrew/core, which support macOS only.

xandrrix@xandrrix-virtual-machine:~$ brew search xmlto
==> Searching local taps...
xmlto                                                                                     xmltoman
==> Searching taps on GitHub...
==> Searching blacklisted, migrated and deleted formulae...
xandrrix@xandrrix-virtual-machine:~$ brew install xmlto
Updating Homebrew...
==> Installing dependencies for xmlto: docbook, docbook-xsl, gettext, gnu-getopt
==> Installing xmlto dependency: docbook
==> Downloading https://linuxbrew.bintray.com/bottles/docbook-5.0.x86_64_linux.bottle.3.tar.gz
######################################################################## 100.0%
==> Pouring docbook-5.0.x86_64_linux.bottle.3.tar.gz
==> xmlcatalog --noout --create /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --del file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/4.2/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --add nextCatalog  file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/4.2/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --del file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/4.1.2/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --add nextCatalog  file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/4.1.2/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --del file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/4.3/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --add nextCatalog  file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/4.3/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --del file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/4.4/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --add nextCatalog  file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/4.4/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --del file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/4.5/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --add nextCatalog  file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/4.5/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --del file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/5.0/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --add nextCatalog  file:///home/xandrrix/.linuxbrew/Cellar/docbook/5.0/docbook/xml/5.0/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> Caveats
To use the DocBook package in your XML toolchain,
you need to add the following to your ~/.bashrc:

export XML_CATALOG_FILES="/home/xandrrix/.linuxbrew/etc/xml/catalog"
==> Summary
🍺  /home/xandrrix/.linuxbrew/Cellar/docbook/5.0: 181 files, 4.7MB
==> Installing xmlto dependency: docbook-xsl
==> Downloading https://linuxbrew.bintray.com/bottles/docbook-xsl-1.79.1.x86_64_linux.bottle.tar.gz
######################################################################## 100.0%
==> Pouring docbook-xsl-1.79.1.x86_64_linux.bottle.tar.gz
==> xmlcatalog --noout --del file:///home/xandrrix/.linuxbrew/Cellar/docbook-xsl/1.79.1/docbook-xsl/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --add nextCatalog  file:///home/xandrrix/.linuxbrew/Cellar/docbook-xsl/1.79.1/docbook-xsl/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --del file:///home/xandrrix/.linuxbrew/Cellar/docbook-xsl/1.79.1/docbook-xsl-ns/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
==> xmlcatalog --noout --add nextCatalog  file:///home/xandrrix/.linuxbrew/Cellar/docbook-xsl/1.79.1/docbook-xsl-ns/catalog.xml /home/xandrrix/.linuxbrew/etc/xml/catalog
🍺  /home/xandrrix/.linuxbrew/Cellar/docbook-xsl/1.79.1: 3,829 files, 90.1MB
==> Installing xmlto dependency: gettext
==> Downloading https://ftp.gnu.org/gnu/gettext/gettext-0.19.8.1.tar.xz
######################################################################## 100.0%
==> ./configure --disable-silent-rules --prefix=/home/xandrrix/.linuxbrew/Cellar/gettext/0.19.8.1  --with-included-glib --with-included-libcroco --with-included-libunistring --wit
==> make
==> make install
🍺  /home/xandrrix/.linuxbrew/Cellar/gettext/0.19.8.1: 1,927 files, 25.2MB, built in 5 minutes 54 seconds
==> Installing xmlto dependency: gnu-getopt
==> Downloading http://frodo.looijaard.name/system/files/software/getopt/getopt-1.1.6.tar.gz
######################################################################## 100.0%
==> make prefix=/home/xandrrix/.linuxbrew/Cellar/gnu-getopt/1.1.6 mandir=/home/xandrrix/.linuxbrew/Cellar/gnu-getopt/1.1.6/share/man install
==> Caveats
This formula is keg-only, which means it was not symlinked into /home/xandrrix/.linuxbrew,
because This formula conflicts with util-linux..

If you need to have this software first in your PATH run:
  echo 'export PATH="/home/xandrrix/.linuxbrew/opt/gnu-getopt/bin:$PATH"' >> ~/.bash_profile

==> Summary
🍺  /home/xandrrix/.linuxbrew/Cellar/gnu-getopt/1.1.6: 34 files, 340.8KB, built in 7 seconds
==> Installing xmlto
==> Downloading https://linuxbrew.bintray.com/bottles/xmlto-0.0.28.x86_64_linux.bottle.1.tar.gz
######################################################################## 100.0%
==> Pouring xmlto-0.0.28.x86_64_linux.bottle.1.tar.gz
🍺  /home/xandrrix/.linuxbrew/Cellar/xmlto/0.0.28: 46 files, 211.9KB
xandrrix@xandrrix-virtual-machine:~$ brew install libpqxx
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
blockhash                dmtx-utils               imagemagick              imagemagick@6            mgba                     pqiv                     zbar

==> Downloading https://github.com/jtv/libpqxx/archive/5.0.1.tar.gz
Already downloaded: /home/xandrrix/.cache/Homebrew/libpqxx-5.0.1.tar.gz
==> ./configure --prefix=/home/xandrrix/.linuxbrew/Cellar/libpqxx/5.0.1_1 --enable-shared
==> make install
Last 15 lines from /home/xandrrix/.cache/Homebrew/Logs/libpqxx/02.make:

*****************************************************

xmlto not found.
Install it, or configure with --disable-documentation

*****************************************************
Makefile:485: recipe for target 'reference-stamp' failed
make[1]: *** [reference-stamp] Error 1
make[1]: *** Waiting for unfinished jobs....
Makefile:505: recipe for target 'tutorial-stamp' failed
make[1]: *** [tutorial-stamp] Error 1
make[1]: Leaving directory '/tmp/libpqxx-20171208-54997-1kh4dop/libpqxx-5.0.1/doc'
Makefile:684: recipe for target 'install-recursive' failed
make: *** [install-recursive] Error 1

READ THIS: https://github.com/Linuxbrew/brew/blob/master/docs/Troubleshooting.md#troubleshooting
Please do not report this issue to Homebrew/brew or Homebrew/core, which support macOS only.

xandrrix@xandrrix-virtual-machine:~$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: Setting LD_* vars can break dynamic linking.
Set variables:
  LD_LIBRARY_PATH: /home/xandrrix/.linuxbrew/lib64:/home/xandrrix/.linuxbrew/lib:

Warning: Homebrew's sbin was not found in your PATH but you have installed
formulae that put executables in /home/xandrrix/.linuxbrew/sbin.
Consider setting the PATH for example like so
  echo 'export PATH="/home/xandrrix/.linuxbrew/sbin:$PATH"' >> ~/.bash_profile

Warning: Homebrew's share was not found in your XDG_DATA_DIRS but you have
this variable set to include other locations.
Some programs like `vapigen` may not work correctly.
Consider setting the XDG_DATA_DIRS for example like so
    echo 'export XDG_DATA_DIRS="/home/xandrrix/.linuxbrew/share:$XDG_DATA_DIRS"' >> ~/.bash_profile
xandrrix@xandrrix-virtual-machine:~$     echo 'export XDG_DATA_DIRS="/home/xandrrix/.linuxbrew/share:$XDG_DATA_DIRS"' >> ~/.bash_profile
xandrrix@xandrrix-virtual-machine:~$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: Setting LD_* vars can break dynamic linking.
Set variables:
  LD_LIBRARY_PATH: /home/xandrrix/.linuxbrew/lib64:/home/xandrrix/.linuxbrew/lib:

Warning: Homebrew's sbin was not found in your PATH but you have installed
formulae that put executables in /home/xandrrix/.linuxbrew/sbin.
Consider setting the PATH for example like so
  echo 'export PATH="/home/xandrrix/.linuxbrew/sbin:$PATH"' >> ~/.bash_profile

Warning: Homebrew's share was not found in your XDG_DATA_DIRS but you have
this variable set to include other locations.
Some programs like `vapigen` may not work correctly.
Consider setting the XDG_DATA_DIRS for example like so
    echo 'export XDG_DATA_DIRS="/home/xandrrix/.linuxbrew/share:$XDG_DATA_DIRS"' >> ~/.bash_profile
xandrrix@xandrrix-virtual-machine:~$     echo 'export XDG_DATA_DIRS="/home/xandrrix/.linuxbrew/share:$XDG_DATA_DIRS"' >> ~/.bash_profile
xandrrix@xandrrix-virtual-machine:~$ echo

xandrrix@xandrrix-virtual-machine:~$   echo

xandrrix@xandrrix-virtual-machine:~$     echo 'export XDG_DATA_DIRS="/home/xandrrix/.linuxbrew/share:$XDG_DATA_DIRS"' >> ~/.bash_profile
xandrrix@xandrrix-virtual-machine:~$ source ~/.bashrc
xandrrix@xandrrix-virtual-machine:~$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: Setting LD_* vars can break dynamic linking.
Set variables:
  LD_LIBRARY_PATH: /home/xandrrix/.linuxbrew/lib64:/home/xandrrix/.linuxbrew/lib:/home/xandrrix/.linuxbrew/lib64:/home/xandrrix/.linuxbrew/lib:

Warning: Homebrew's sbin was not found in your PATH but you have installed
formulae that put executables in /home/xandrrix/.linuxbrew/sbin.
Consider setting the PATH for example like so
  echo 'export PATH="/home/xandrrix/.linuxbrew/sbin:$PATH"' >> ~/.bash_profile

Warning: Homebrew's share was not found in your XDG_DATA_DIRS but you have
this variable set to include other locations.
Some programs like `vapigen` may not work correctly.
Consider setting the XDG_DATA_DIRS for example like so
    echo 'export XDG_DATA_DIRS="/home/xandrrix/.linuxbrew/share:$XDG_DATA_DIRS"' >> ~/.bash_profile
xandrrix@xandrrix-virtual-machine:~$   echo 'export PATH="/home/xandrrix/.linuxbrew/sbin:$PATH"' >> ~/.bash_profile
xandrrix@xandrrix-virtual-machine:~$ source ~/.bashrc
xandrrix@xandrrix-virtual-machine:~$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: Setting LD_* vars can break dynamic linking.
Set variables:
  LD_LIBRARY_PATH: /home/xandrrix/.linuxbrew/lib64:/home/xandrrix/.linuxbrew/lib:/home/xandrrix/.linuxbrew/lib64:/home/xandrrix/.linuxbrew/lib:/home/xandrrix/.linuxbrew/lib64:/home/xandrrix/.linuxbrew/lib:

Warning: Homebrew's sbin was not found in your PATH but you have installed
formulae that put executables in /home/xandrrix/.linuxbrew/sbin.
Consider setting the PATH for example like so
  echo 'export PATH="/home/xandrrix/.linuxbrew/sbin:$PATH"' >> ~/.bash_profile

Warning: Homebrew's share was not found in your XDG_DATA_DIRS but you have
this variable set to include other locations.
Some programs like `vapigen` may not work correctly.
Consider setting the XDG_DATA_DIRS for example like so
    echo 'export XDG_DATA_DIRS="/home/xandrrix/.linuxbrew/share:$XDG_DATA_DIRS"' >> ~/.bash_profile

**Comment by iMichka** [comment]
Hi. Installing `xmlto` alone is not sufficient. It needs to be added as an explicit dependency in the formula.

Could you try adding this line to the formula: `depends_on "xmlto" => :build unless OS.mac?` ?

A pull request would be welcome if this works for you.

**Comment by xandrrix** [comment]
Thank you so much Sir iMichka! SOLVED just like that. I'm not sure how to push that formula change to the repo for everyone else to use, but I've pasted it below. But I'll poke around and see if I can't figure out how to actually pull/update as you asked. Cheers! 

Final version of formula was:
```
class Libpqxx < Formula
  desc "C++ connector for PostgreSQL"
  homepage "http://pqxx.org/development/libpqxx/"
  url "https://github.com/jtv/libpqxx/archive/5.0.1.tar.gz"
  sha256 "21ba7167aeeb76142c0e865127514b4834cefde45eaab2d5eb79099188e21a06"
  revision 1

  bottle do
    cellar :any
    sha256 "4a146fd2c38fed728cfbc51502c9976953cf15e0a278e5d0b0063006b6feedf9" => :high_sierra
    sha256 "e09f197b4b6d84212cbe09a5c71b6ebab248a1e634c81377d6b8d7172cb251ed" => :sierra
    sha256 "14cabe29c05d71602958b86c775681a9dc530ba269378a5a2c41737b8aa80b1f" => :el_capitan
  end

  depends_on "doxygen" => :build unless OS.mac?
  depends_on "xmlto" => :build unless OS.mac?
  depends_on "pkg-config" => :build
  depends_on :postgresql

  def install
    system "./configure", "--prefix=#{prefix}", "--enable-shared"
    system "make", "install"
  end

  test do
    (testpath/"test.cpp").write <<~EOS
      #include <pqxx/pqxx>
      int main(int argc, char** argv) {
        pqxx::connection con;
        return 0;
      }
    EOS
    system ENV.cxx, "test.cpp", "-L#{lib}", "-lpqxx", "-I#{include}", "-o", "test"
    # Running ./test will fail because there is no runnning postgresql server
    # system "./test"

    # `pg_config` uses Cellar paths not opt paths
    postgresql_include = Formula["postgresql"].opt_include.realpath.to_s
    assert_match postgresql_include, (lib/"pkgconfig/libpqxx.pc").read,
                 "Please revision bump libpqxx."
  end
end

```

**Comment by iMichka** [comment]
Cool. There are some instructions to get you started: https://github.com/Linuxbrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md

I can guide you through the process if you have any questions.

**Comment by iMichka** [comment]
Pull request opened here: #5328

**Comment by iMichka** [comment]
Fixed by #5328
